scripts/ecs-anywhere-install.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -e # exit on failure
+umask 022
```

Comment:
is this a necessary part of this fix? If i understand correctly I would think that we only need one of `chmod` and `umask` ?

Reply:
I would generally prefer if the fix for this one file didn't have to change the permissions of every file created by this script. Could we chmod just the deb file after downloading it to `755`?

Reply:
The intent was to put some control around the environment because we don't know what the current value of the `umask` is. If it is `077` then all files created in this script will be readable/writable only by the `root` user. This has adverse consequences for the `apt install` later on because it needs to run as user `_apt`. 

If you're concerned about adverse side effects (can you think of any?) then it's not harmful to remove this and `chmod` the downloaded files instead as needed.